phantombuster/web-browser-extension
# Add connect button to session cookie modal

## 📝 What is this PR
[CP-450](https://phantombuster.atlassian.net/jira/software/projects/CP/boards/46?selectedIssue=CP-450)

[Design specs](https://www.figma.com/file/yVP49ptOKD6lqlqLJiceRs/Flow-Builder---Edition-V1.0?node-id=1-4625&t=TvImOk5oCGuJp6cu-4)

I am updating the extension code because the `Connect to LinkedIn` button must now be rendered on the Agent console page.

## 🧪 QA Process
- Build the extension using the development mode (`npm run build:dev`)
- Go to this preview => https://3227-buster.preview.phantomcompany.io/
- Login with my account with :`karim.riahi@phantombuster.com` (password is `asdfasdf`)
- navigate to the console page of this Agent => https://3227-buster.preview.phantomcompany.io/7757777578095235/phantoms/6994644860412626/dashboard
- Click on the `Resolve` button. You should see a "Connect to LinkedIn" button in the edit session cookie modal.
![image](https://user-images.githubusercontent.com/12436800/232999096-ea9a3f46-0d40-4b76-a0f0-79cfa494a322.png)
- Clicking on the button should fill the session cookie field.
